dashboard/app/models/ability.rb

```diff
@@ -320,7 +320,18 @@ def initialize(user)
     # In order to accommodate the possibility of there being no database, we
     # need to check that the user is persisted before checking the user
     # permissions.
-    if user.persisted? && user.permission?(UserPermission::LEVELBUILDER)
+
+    # When in levelbuilder_mode, we want to grant users with levelbuilder
+    # permissions broad abilities to change curriculum and form objects.
+    #
+    # Note: We also grant these abilities in the 'test' environment to support
+    # running UI tests that cover level editing without having levelbuilder_mode
+    # set. An unfortunate side effect of this is that unit tests that cover the
+    # levelbuilder permission will mimic levelbuilder_mode instead of production
```

Comment:
Our UI and eyes tests use levelbuilder permissions. Will these be affected? Example: https://github.com/code-dot-org/code-dot-org/blob/b527938e684a8442454e632ce64d988f992c3e00/dashboard/test/ui/features/javalab/neighborhood.feature#L7

Reply:
Thanks for the heads up, I will keep an eye on it.  There should be no change for the javabuilder_session ability because of the added check on line 382.